### PR TITLE
Remove Sphinx as a required dependency

### PR DESCRIPTION
**Pull request by phlax**

### CODE REVIEW

**Comment by andy-maier** [comment]
Gentle reminder to get this in. We are also hampered by the circular dependency (using pipdeptree).

File: CHANGES [hunk 1]
```diff
@@ -1,3 +1,8 @@
+Release 1.0.6 (Pending)
+==========================
```
Comment (AA-Turner) [inline CHANGES:2]:
```suggestion
Release 1.0.6 (unreleased)
==========================
```

File: pyproject.toml [hunk 2]
```diff
@@ -53,6 +50,9 @@ lint = [
     "mypy",
     "docutils-stubs",
 ]
+standalone = [
+  "sphinx > 5",
```
Comment (AA-Turner) [inline pyproject.toml:54]:
```suggestion
    "Sphinx>=5",
```

File: CHANGES [hunk 3]
```diff
@@ -1,3 +1,8 @@
+Release 1.0.6 (unreleased)
+==========================
+
+* Fix circular dependency with Sphinx that caused failure in DAG-based package management
```
Comment (AA-Turner) [inline CHANGES:4]:
```suggestion
* Remove Sphinx as a required dependency, as circular dependencies may cause
  failure with package managers that expect a directed acyclic graph (DAG)
  of dependencies.
```

File: pyproject.toml [hunk 4]
```diff
@@ -39,9 +39,6 @@ classifiers = [
     "Topic :: Text Processing",
     "Topic :: Utilities",
 ]
-dependencies = [
-    "Sphinx>=5",
-]
 dynamic = ["version"]
```
Comment (AA-Turner) [inline pyproject.toml:42]:
```suggestion
dependencies = []
dynamic = ["version"]
```

**Pull request merged**